src/main/java/org/jenkinsci/plugins/github/internal/GitHubClientCacheOps.java

```diff
@@ -94,6 +95,7 @@ public static Path getBaseCacheDir() {
      *
      * @param configs active server configs to exclude caches from cleanup
      */
+    @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE")
```

Comment:
hey, can you give more insights of why is this necessary?

Reply:
@lanwen 
Spotbugs continues to have issues with properly identifying the the null check automatically generated when using [try with resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) as something that cannot be fixed by the person writing the code.  It reports a problem with [line 109](https://github.com/jenkinsci/github-plugin/pull/228/files#diff-92e6819e3ec96cf3b77dec0a1aece0deR109) which isn't even changed in this PR.